CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+
+### Writing a Good Bug Report
+
+Writing good bug reports make it easier for maintainers to verify and identify the root cause of the problem. The better the bug report, the faster the problem can be resolved. A bug report should contain the following information:
+
+- **Description of the Bug**: What's the issue you encountered?
+- **Reproduction Steps**: How can the issue be reproduced?
+- **Expected Behaviour**: What do you expect to happen?
+- **Information on the environment**: OS, Browser, Device
+- **Additional Notes**: Add any other notes or screenshots about the bug here.
+
+When you are ready to submit a bug report, please use the [Bug Report issue template](https://github.com/compsci-adl/website/issues/new?assignees=&labels=&projects=&template=bug-report.yml).
+
+### Writing a Good Feature Request
+
+Writing good feature requests make it easier for maintainers to understand the feature and see how it fits into the website. The better the feature request, the faster the feature can be discussed and possibly implemented. A feature request should contain the following information:
+
+- **Overview**: Include the basic, high-level concepts for this feature here.
+- **Details**: These may include specific methods of implementation, design considerations, or any other technical details.
+- **Why would this feature be useful?**: A clear and concise description of why this feature would improve the CS Club website.
+- **Additional Notes**: Add any other notes or screenshots about the feature request here.
+
+When you are ready to submit a request, please use the [Feature Request issue template](https://github.com/compsci-adl/website/issues/new?assignees=&labels=&projects=&template=feature-request.yml).
+
+## Contributing Changes
+
+Project maintainers will merge changes that both improve the project and meet our code quality standards. 
+
+### Suggested Workflow
+
+We recommend you to use the following workflow:
+
+1. Find an existing issue or create a new one.
+    - Seek feedback from the maintainers and the community that your proposed change is a good one.
+    - Clearly state whether you plan on implementing it yourself or not. If this is the case, you can request that the issue be assigned to you. Note: The person who filed the issue and the person who implements it don't have to be the same.
+2. Create a personal fork of the repository using GitHub (if you don't already have one).
+3. In your fork, create a branch based of main (`git checkout -b mybranch`).
+4. Create and commit your changes to your branch.
+    - Commit messages should follow the guidelines in [Commit Messages](#commit-messages)
+6. Run the linting and prettier commands with `pnpm lint` and `pnpm format` to ensure your code follows our code quality standards.
```

Comment:
The commands should be `pnpm run lint` and `pnpm run format`. Although as far as I know, there is a git hook that does both automatically when pushing to GitHub? Am I correct @jsun969?

Reply:
I think he implemented the git hook for it, but at least for me, it doesn't seem to work properly since I have to run them manually for it to check the code.

Reply:
We have format githook

Reply:
When you lint, some warnings don't get automatically fixed, so it's beneficial to manually lint as well.